### ISSUE TITLE
Name the runner website??

### ISSUE THREAD

**Issue by spectranaut** [w3c/aria-at#105]
We should have a cool name for the production runner!

**Comment by spectranaut** [comment]
Because they are birds that run (so, they are "runners") @evmiguel and I have been calling the runner "[Emu](https://en.wikipedia.org/wiki/Emu)".

**Comment by zcorpan** [comment]
I think it should be super clear what it is, at least if you know about the project. The accessibility community seems to have a lot of acronyms and clever names (like "Silver"), and it's not very accessible or inclusive, ironically.

**Comment by jongund** [comment]
I agree, I prefer a name that people not associated with the project would have some idea about what it is for.   But that sometimes can get wordy and defeats the purpose of a short name.

One potential name is:
AT Runner short for Assistive Technology with ARIA Runner

**Comment by zcorpan** [comment]
ARIA-AT Runner?
